.gitignore

```diff
@@ -0,0 +1,32 @@
+# OSX leaves these everywhere on SMB shares
+._*
+
+# OSX trash
+.DS_Store
+
+# Eclipse files
+.classpath
+.project
+.settings/**
+
+# Files generated by JetBrains IDEs, e.g. IntelliJ IDEA
+.idea/
+*.iml
+
+# Vscode files
+.vscode
+
+# Emacs save files
+*~
+\#*\#
+.\#*
+
+# Vim-related files
+[._]*.s[a-w][a-z]
+[._]s[a-w][a-z]
+*.un~
+Session.vim
+.netrwhist
+
+# User cluster configs
+.kubeconfig
```

Comment:
Hey! I think that's better add the other files in[this](https://github.com/kubernetes/kubernetes/blob/master/.gitignore) gitignore, can be useful in future.

Reply:
If these are going to be required in the future, then sure, I will add them :)

Reply:
@Matheusafonsouza Most of those files in k/k are repo specific. So agree with @praveenrewar that we could add them later in a separate PR. The current file is a good starting point.

Reply:
@PushkarJ Okay! Thanks :)